### PR TITLE
Fix grammar and spelling errors

### PR DESCRIPTION
**Pull request by donno2048**

### CODE REVIEW

**Comment by Angeetty** [comment]
Bonsoir.
Dans un premier temps les histoires de correction d'orthographe m'intéresse beaucoup

**Comment by Angeetty** [comment]
Cherche un homme avec qui travailler

File: README.md [hunk 1]
```diff
@@ -241,9 +241,9 @@ following rules are enabled by default:
 * `git_not_command` &ndash; fixes wrong git commands like `git brnch`;
 * `git_pull` &ndash; sets upstream before executing previous `git pull`;
 * `git_pull_clone` &ndash; clones instead of pulling when the repo does not exist;
-* `git_pull_uncommitted_changes` &ndash; stashes changes before pulling and pops them afterwards;
+* `git_pull_uncommitted_changes` &ndash; stashes changes before pulling and pops them afterward;
```
Comment (scorphus) [inline README.md:244]: "afterwards" is a correct usage, if I'm not mistaken.
```suggestion
* `git_pull_uncommitted_changes` &ndash; stashes changes before pulling and pops them afterwards;
```

**Review by scorphus — APPROVED** [review]
Thanks for the corrections. Please, observe my only suggestion.

File: README.md (hunk 1 above)
Comment (donno2048) [reply README.md:244]: Are you Canadian?

Comment (scorphus) [reply README.md:244]: Maybe I am (and probably the the athor of the original text is too) more used to British English 😊

Comment (donno2048) [reply README.md:244]: Seems legit, I'm just used to American

**Review by scorphus — APPROVED** [review]
Thanksm @donno2048

**Pull request merged**